js/src/personal/kazi/Kazi.page.tsx

```diff
@@ -1,18 +1,29 @@
-import { Title, Text } from '@mantine/core'
+import { Title, Text, useMantineColorScheme } from '@mantine/core'
 import React, { useState } from 'react'
 import { FaSun, FaMoon } from 'react-icons/fa'
 import styles from './Kazi.module.css'
 
 export function KaziPage() {
   const [sun, setSun] = useState(true)
-  const [sunAnimate, setSunAnimate] = useState(false)
+  const [animate, setAnimate] = useState('')
+  const { setColorScheme } = useMantineColorScheme()
 
-  const sunToMoon = () => {
-    setSunAnimate(true)
-    setTimeout(() => {
-      setSun(!sun)
-      setSunAnimate(false)
-    }, 2000)
+  const changeIcon = () => {
+    if (animate === '') {
+      setAnimate(sun ? 'exitSun' : 'exitMoon')
+      setTimeout(() => {
+        setSun(!sun)
+        setAnimate(sun ? 'enterMoon' : 'enterSun')
+        setTimeout(() => {
+          setAnimate('')
+        }, 750)
+      }, 750)
+    }
+    if (!sun) {
+      setColorScheme('light')
```

Comment:
is `setColorScheme(sun ? 'dark' : 'light')` cleaner?

Reply:
When I first tried it, it didn't work, could be where I placed it earlier but it works now. Will be setting the color scheme in this way.